wiki/osu!_tournament_client/en.md

```diff
@@ -1,27 +1,27 @@
-# osu!lazer tournament client
+# osu! tournament client
 
-**osu!lazer tournament client** is the official client that overlays [osu!tourney](/wiki/osu!tourney) with supplementary scenes and information relevant to an osu! tournament during a live stream.
+The **osu! tournament client** is the official client that overlays [osu!tourney](/wiki/osu!tourney) with supplementary scenes and information relevant to an osu! tournament during a live stream.
 
 Users encountering problems with the client may [create an issue on GitHub](https://github.com/ppy/osu/issues) or send an email to [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
 
 ## Setup
 
-To start the osu!lazer tournament client, you need to specify a launch argument to the osu!lazer executable. To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%/osulazer/osu!.exe --tournament`. This will make that specific shortcut start up osu!lazer in the tournament client mode.
+To start the osu! tournament client, you need to specify a launch argument to the osu!(lazer) executable. To do this, create a new shortcut on the desktop and set the location for it to `%LOCALAPPDATA%/osulazer/osu!.exe --tournament`. This will make that specific shortcut start up osu!(lazer) in the tournament client mode.
```

Comment:
> `%LOCALAPPDATA%/osulazer/`

:|

Reply:
This is a bit of an unfortunate one yeah. `osu!` is already taken so we can't reuse it here.

![Parallels Desktop 2022-02-21 at 07 26 31](https://user-images.githubusercontent.com/191335/154908056-15c07a5d-55f8-4f4b-9016-c995dc39e86f.png)

It's on my list of things we need to revisit alongside improving the migration and update process.